pom.xml

```diff
@@ -273,6 +273,11 @@
       <artifactId>unused</artifactId>
       <version>1.0.0</version>
     </dependency>
+    <dependency>
```

Comment:
No, generally speaking the parent POM does not declare dependencies (there are some odd exceptions here). The actual modules declare dependencies, and not every single one needs scala-library I think. The other modules already declare a dependency on scala-library, like core; tags isn't how they get it. what is the problem that you're solving?

Reply:
@srowen Thanks! I got it!I will close this PR.